helper/communicator/config.go

```diff
@@ -167,6 +167,37 @@ type SSH struct {
 	SSHPrivateKey []byte `mapstructure:"ssh_private_key"`
 }
 
+type WinRM struct {
```

Comment:
just moving this up so it could be together with the other structs

Reply:
Nice !